src/pages/MembersPage/MembersList/MembersList.jsx

```diff
@@ -40,11 +39,11 @@ function useActivateUser({ provider, owner }) {
   return { activate, ...rest }
 }
 
-const handleActivate = (accountDetails, activate, setIsOpen) => (user) => {
+const handleActivate = (planData, activate, setIsOpen) => (user) => {
   if (
-    accountDetails?.activatedUserCount >= accountDetails?.plan?.quantity &&
+    !planData?.plan?.hasSeatsLeft &&
```

Comment:
We're using hasSeatsLeft in favor of accountDetails business logic here -- cleaning up the code a bit imo :)